undertow/src/main/java/org/wildfly/extension/undertow/UndertowSubsystemSchema.java

```diff
@@ -34,10 +39,11 @@ public enum UndertowSubsystemSchema implements PersistentSubsystemSchema<Underto
     VERSION_11_0(11),   // WildFly 20-22    N.B. There were no parser changes between 10.0 and 11.0 !!
     VERSION_12_0(12),   // WildFly 23-26.1, EAP 7.4
     VERSION_13_0(13),   // WildFly 27       N.B. There were no schema changes between 12.0 and 13.0!
-    VERSION_14_0(14),   // WildFly 28-present
+    VERSION_14_0(14),   // WildFly 28
+    VERSION_15_0_PREVIEW(15, 0, Stability.PREVIEW)   // WildFly 32-present
```

Comment:
I think the new schema version should be `Version_14_preview`, `version_14_community` etc. 
If you add another preview schema once this RFE is released, then the new one would be `15:preview`